### ISSUE TITLE
Project not working in eclipse.

### ISSUE THREAD

**Issue by ghost** [ChaitanyaYeole/EasyResume-Java#1]
I downloaded this project for taking an idea but there is an issue with the loginServlet.java file which is i think the file you have to run in order to start the project, kindly help me resolve the issue.

**Comment by ChaitanyaYeole** [comment]
> I downloaded this project for taking an idea but there is an issue with the loginServlet.java file which is i think the file you have to run in order to start the project, kindly help me resolve the issue.

No, You have to run index.jsp file, as it is main start of project.
Hope this will fix your issue.
